test/odbc/psqlodbc/test_pg_ntext.cpp

```diff
@@ -0,0 +1,382 @@
+   }
+
+};
+
+TEST_F(PSQL_DataTypes_Ntext, Table_Creation) {
+  const vector<int> LENGTH_EXPECTED = {4, 8190};
+  const vector<int> PRECISION_EXPECTED = {0, 0};
+  const vector<int> SCALE_EXPECTED = {0, 0};
+  const vector<string> NAME_EXPECTED = {"int4", "text"};
+  const vector<string> PREFIX_EXPECTED = {"int4", "'"};
+  const vector<string> SUFFIX_EXPECTED = {"int4", "'"};
+
+  const vector<int> IS_CASE_SENSITIVE = {0, 1};
+
+  createTable(ServerType::PSQL, TABLE_NAME, TABLE_COLUMNS);
+  testCommonCharColumnAttributes(ServerType::PSQL, TABLE_NAME, TABLE_COLUMNS.size(), COL1_NAME, LENGTH_EXPECTED, 
+    PRECISION_EXPECTED, SCALE_EXPECTED, NAME_EXPECTED, IS_CASE_SENSITIVE, PREFIX_EXPECTED, SUFFIX_EXPECTED);
+  dropObject(ServerType::PSQL, "TABLE", TABLE_NAME);
+}
+
+TEST_F(PSQL_DataTypes_Ntext, Table_Create_Fail) {
+  const vector<vector<pair<string, string>>> invalid_columns {
+    {{"invalid1", DATATYPE + "(4)"}} // Cannot specify a column width on data type datatime.
+  };
+
+  // Assert that table creation will always fail with invalid column definitions
+  testTableCreationFailure(ServerType::PSQL, TABLE_NAME, invalid_columns);
+  dropObject(ServerType::PSQL, "TABLE", TABLE_NAME);
+}
+
+// // inserted values differ that of expected?
+TEST_F(PSQL_DataTypes_Ntext, Insertion_Success) {
+  const vector<string> INSERTED_VALUES = {
+    "NULL", // NULL value
+    STRING_1,
+    STRING_4000,
+    STRING_20,
+    "" // blank value
+  };
+
+  const vector<string> expected = {
+    "NULL", // NULL value
+    STRING_1,
+    STRING_4000,
+    STRING_20,
+    "" // blank value
+  };
+
+  createTable(ServerType::PSQL, TABLE_NAME, TABLE_COLUMNS);
+  testInsertionSuccess(ServerType::PSQL, TABLE_NAME, COL1_NAME, INSERTED_VALUES, expected);
+  dropObject(ServerType::PSQL, "TABLE", TABLE_NAME);
+}
+
+TEST_F(PSQL_DataTypes_Ntext, DISABLED_Insertion_Failure) {
+  const vector<string> INSERTED_VALUES = {
+    STRING_4000 + "t"
+  };
+
+  createTable(ServerType::PSQL, TABLE_NAME, TABLE_COLUMNS);
+  testInsertionFailure(ServerType::PSQL, TABLE_NAME, COL1_NAME, INSERTED_VALUES, false);
+  dropObject(ServerType::PSQL, "TABLE", TABLE_NAME);
+}
+
+TEST_F(PSQL_DataTypes_Ntext, Updata_Success) {
+  const vector<string> INSERTED_VALUES = {
+    "a"
+  };
+
+  const vector<string> UPDATED_VALUES = {
+    "NULL",
+    STRING_1,
+    STRING_20,
+    STRING_4000,
+    "" // blank value
+  };
+
+  const vector<string> expected_UPDATED_VALUES = {
+    "NULL",
+    STRING_1,
+    STRING_20,
+    STRING_4000,
+    "" // blank value
+  };
+
+  createTable(ServerType::PSQL, TABLE_NAME, TABLE_COLUMNS);
+  testInsertionSuccess(ServerType::PSQL, TABLE_NAME, COL1_NAME, INSERTED_VALUES, INSERTED_VALUES);
+  testUpdateSuccess(ServerType::PSQL, TABLE_NAME, COL1_NAME, COL2_NAME, UPDATED_VALUES, expected_UPDATED_VALUES);
+  dropObject(ServerType::PSQL, "TABLE", TABLE_NAME);
+}
+
+TEST_F(PSQL_DataTypes_Ntext, DISABLED_Updata_Fail) {
```

Comment:
Ditto to the comment on `Insertion_Fail` & Typo in test name
```suggestion
TEST_F(PSQL_DataTypes_Ntext, DISABLED_Update_Fail) {
```